opencti-platform/opencti-front/src/private/components/nav/TopMenuSettings.js

```diff
@@ -35,7 +39,7 @@ const TopMenuSettings = () => {
                 classes={{ root: classes.button }}>
                 {t('Parameters')}
             </Button>
-            <Security needs={[SETTINGS_SETACCESSES]}>
+            <Security needs={[SETTINGS_SETMARKINGS]}>
```

Comment:
I don't understand this security.
If I don't have the SETTINGS_SETMARKINGS, so the button is not display, but the others routes inside has not always the security SETTINGS_SETMARKINGS (exemple: dashboard/settings/accesses/roles)

So I don't be bale to go to roles if I don't have the security for marking ?

Reply:
Right, we missed some stuff and it's not working that way. I'll correct it